samples/Makefile

```diff
@@ -105,8 +108,11 @@ EXELIST	=	ascii		\
 # --------------------------------------------------------------------------
 # Rules to make the binaries
 
-.PHONY:	all
-all:	$(EXELIST)
+.PHONY: all samples
+all:
+
+samples:
+	$(EXELIST)
```

Comment:
We missed a couple of things:

- `$(EXELIST)` must be prerequisites, not a recipe.
- The prerequisite for `samples.d64` must be changed to `samples`.